scripts/haml.rb

```diff
@@ -1,6 +1,15 @@
 require 'rubygems'
 require 'bundler/setup'
 require 'haml'
+require 'rails_i18n'
+require 'action_view'
+
+# Allow shorthand t() helper to be used
+include ActionView::Helpers::TranslationHelper
+
+I18n.available_locales = [:en, :cy]
+I18n.load_path += Dir['locales/*.yml']
```

Comment:
AFAIK on the Rails / consumer side we'd need to do a similar thing and include `@citizensadvice/design-system/locales`  in the load path.

Reply:
This is a very elegant approach

Reply:
To confirm I've tested this via `npm link` and adding `config.i18n.load_path += Dir[Rails.root.join("node_modules/@citizensadvice/design-system/locales/*.yml")]` to the application config. It's able to load design-system bundled locales in addition to any app ones.

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/123386/95088686-2cad5300-071b-11eb-8fe3-843a0e890c0f.png">